test/testdrive/subscribe-output.td

```diff
@@ -0,0 +1,255 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ set-regex match=\d{13,20} replacement=<TIMESTAMP>
+
+$ postgres-connect name=mz_system url=postgres://mz_system:materialize@${testdrive.materialize-internal-sql-addr}
+$ postgres-execute connection=mz_system
```

Comment:
Testdrive runs with all feature flags enabled, so there is no need to explictily enable them here.

However, you can consider adding a section to `test/sql-feature-flags/mzcompose.py` -- it is a framework that tests that a given flag causes an error to be or not be returned, as appropriate.

Reply:
I run the test on the command line using 
```
env COCKROACH_URL=postgres://root@localhost:26257 cargo run --bin testdrive -- test/testdrive/subscribe-output.td --default-timeout=1s
```
and if i don't set these flags i get an error.

Is there a better command? I have noticed that all feature flags are enabled with running precommit tests

Reply:
You need to start environmentd with `--unsafe-mode` the way it is done in the CI

Reply:
ah, thanks!